stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/ChatSocketServiceImpl.kt

```diff
@@ -59,7 +63,10 @@ internal class ChatSocketServiceImpl constructor(
         }
 
         override fun onDisconnected() {
-            state = State.NetworkDisconnected
+            healthMonitor.stop()
```

Comment:
As far as we don't have Network Connection, doesn't make sense to have `healthMonitor` working on the background